src/Components/CreateImageWizard/ValidatedTextInput.tsx

```diff
@@ -90,6 +145,57 @@ export const HookValidatedInput = ({
   );
 };
 
+export const HookValidatedInputWithPasswordVisibilityButton = (
```

Comment:
![image](https://github.com/user-attachments/assets/c45fb479-b70b-4f74-b7b9-38d4e6b02649)

I think we should try reusing the <HookValidatedInput> component inside of the <Password> component, extending <HookValidatedInput> if necessary.